src/test/java/org/mockbukkit/mockbukkit/inventory/ItemStackMockTest.java

```diff
@@ -687,4 +690,27 @@ void removeEnchantments()
 		assertTrue(item.getEnchantments().isEmpty());
 	}
 
+	@Test
+	void getDisplayName()
+	{
+		ItemStackMock item = new ItemStackMock(Material.DIAMOND_PICKAXE);
+		Component displayName1 = item.displayName();
+		assertNotNull(displayName1);
+		assertEquals("[Diamond Pickaxe]", PlainTextComponentSerializer.plainText().serialize(displayName1));
+		ItemMeta meta = item.getItemMeta();
+		meta.customName(Component.text("Hello world!"));
+		item.setItemMeta(meta);
+		Component displayName2 = item.displayName();
+		assertNotNull(displayName2);
+		assertNotEquals(displayName1, displayName2);
+		assertEquals("[Hello world!]", PlainTextComponentSerializer.plainText().serialize(displayName2));
```

Comment:
Are this brackets normal?

Reply:
I mostly guessed that from the code in paper. But, yeah probably a good idea to doube check that against a real instance

Reply:
@4everTheOne Yep. I just tested this, and this is the case. So the test is correct